va/va_dec_hevc.h

```diff
@@ -561,6 +565,22 @@ typedef struct  _VASliceParameterBufferHEVCExtension
     VASliceParameterBufferHEVCRext           rext;
 } VASliceParameterBufferHEVCExtension;
 
+/**
+ * \brief HEVC Decoding Subsets Parameter Buffer Structure For Long Format
+ *
+ * The data layout on the slice subsets parameter buffer is concatenation with one or multiple
+ * subset entry point offsets. All the offset values are layed out one by one according to slice
+ * order with first slice segment first, second slice segment second, etc... The total entry number
+ * is indicated by parameter TotalNumEntryPointOffsets(no definition) in VAPictureParameterBufferHEVC.
+ * And the first entry position of the entry point offsets for any slice segment is indicated by
+ * parameter entry_offset_to_subset_array in VAPictureParameterBufferHEVC data structure.
+ */
+typedef struct _VASubsetsParameterBufferHevc
+{
+    /** HEVC syntaxt element. */
+    uint32_t entry_point_offset_minus1[440]; 
+} VASubsetsParameterBufferHevc;
```

Comment:
this structure will be a stable one ? will not changed in future, right?

Reply:
@XinfengZhang yeah, will not change.